src/metabase/models/revision/diff.clj

```diff
@@ -1,50 +1,76 @@
 (ns metabase.models.revision.diff
   (:require
    [clojure.core.match :refer [match]]
-   [clojure.string :as str]))
+   [metabase.util.i18n :refer [deferred-tru]]
+   [toucan2.core :as t2]))
 
-(defn- diff-string* [k v1 v2]
+(defn- diff-strings* [k v1 v2 identifier]
```

Comment:
this logic try to follow https://github.com/metabase/metabase/blob/d4621d141e441f8fdd8bf16de698e5197016a1ef/frontend/src/metabase/lib/revisions/revisions.js#L100

Reply:
Still waiting for this to be resolved, in case you missed

Reply:
I already renamed it with your other suggestion, here is the commit https://github.com/metabase/metabase/pull/30502/commits/4f1a002a091e945a797ca327c0fd78fa64e9ef83

Reply:
Of course, my bad.